docs/features-timeline.md

```diff
@@ -0,0 +1,251 @@
+---
+id: features-timeline
+title: Features Timeline
+---
+
+
+<ol class="timeline-container">
+<li data-date="Feb 2021">
+
+## Babel 7.13.0
+
+[blog post](https://babeljs.io/blog/2021/02/22/7.13.0)
+
+- Top-level `targets` option ([RFC](https://github.com/babel/rfcs/pull/2))
```

Comment:
can link to https://babeljs.io/docs/en/options#output-targets

Reply:
```suggestion
- Top-level [`targets`](https://babeljs.io/docs/en/options#output-targets) option ([RFC](https://github.com/babel/rfcs/pull/2))
```